src/main/java/gregtech/common/blocks/BlockGregStairs.java

```diff
@@ -0,0 +1,14 @@
+package gregtech.common.blocks;
+
+import gregtech.api.GregTechAPI;
+import net.minecraft.block.BlockStairs;
+import net.minecraft.block.state.IBlockState;
+
+public class BlockGregStairs extends BlockStairs {
+
+    public BlockGregStairs(IBlockState state) {
+        super(state);
+        setCreativeTab(GregTechAPI.TAB_GREGTECH);
+        this.useNeighborBrightness = true;
```

Comment:
Why does this need to be set?

Reply:
Not setting the flag makes light behave strangely. You can see the field being automatically set for vanilla blocks during registration.